alea/model.py

```diff
@@ -318,19 +376,22 @@ def confidence_interval(
             confidence_interval_kind: str = None,
             **kwargs) -> Tuple[float, float]:
         """
-        Uses self.fit to compute confidence intervals for a certain named parameter
-
-        poi_name: string, name of fittable parameter of the model
-        parameter_interval_bounds: range in which to search for the confidence interval edges.
-            May be specified as:
-            - setting the property "parameter_interval_bounds" for the parameter
-            - passing a list here
+        Uses self.fit to compute confidence intervals for a certain named parameter.
         If the parameter is a rate parameter, and the model has expectation values implemented,
-        the bounds will be interpreted as bounds on the expectation value
-        (so that the range in the fit is parameter_interval_bounds/mus)
-        otherwise the bound is taken as-is.
+        the bounds will be interpreted as bounds on the expectation value,
```

Comment:
Ok very minor detail but start with large T here.

Reply:
It follows a line that ends with `,`.